bundles/org.eclipse.ui.ide/extensions/org/eclipse/ui/actions/NewWizardMenu.java

```diff
@@ -186,9 +191,28 @@ protected void addItems(List<IContributionItem> list) {
 			list.add(new ActionContributionItem(newExampleAction));
 			list.add(new Separator());
 		}
+		// To add shortcuts from OTHER... wizard regardless of perspective
+		Collection<IContributionItem> otherItems = new LinkedList<>();
+		if (!RecentNewWizardSelection.getInstance().getSelectedFromOther().isEmpty()) {
```

Comment:
Method name `getSelectedFromOther()` and derived variable names are unfortunate, I had no idea how to understand these names.
Why not just `RecentNewWizardSelection.getInstance().getItems()` ?